posthog/api/dashboard.py

```diff
@@ -150,8 +162,15 @@ def update(self, instance: Dashboard, validated_data: Dict, *args: Any, **kwargs
         if validated_data.get("is_shared") and not instance.share_token:
             instance.share_token = secrets.token_urlsafe(22)
 
+        # TODO
```

Comment:
@mariusandra what was this TODO to, erm, do?

Reply:
It's just sittin' on the dock of the bay...

(can be removed I guess)